app/views/content/international-teacher-training-to-english-standards-introducing-iqts.md

```diff
@@ -0,0 +1,177 @@
+---
+title: International qualified teacher status (iQTS)
+description: |-
+ International qualified teacher status (iQTS) is a new teaching qualification backed by England’s Department for Education (DfE). iQTS is equivalent to qualified teacher status (QTS). You can train where you live and work. 
+date: "2021-08-06"
+image: "media/images/content/hero-images/0003.jpg"
+backlink: "../"
+keywords: 
+    - international
+    - QTS
+    - iQTS
+    - Qualified teacher status
+---
+
+## Introducing iQTS
+
+International qualified teacher status (iQTS) is a new teaching qualification backed by England’s Department for Education (DfE). iQTS is equivalent to qualified teacher status (QTS). You can train where you live and work. 
+
+## Key facts
+
+1. Apply now for courses starting September 2022
+2. Train for a year via remote learning and at a school in your region 
```

Comment:
I might split point 2 into two separate points just for clarity.

2. Training for your iQTS takes a year
3. The course includes remote and school-based learning